app/views/layouts/_sidebar.html.erb

```diff
@@ -0,0 +1,16 @@
+<div id="sidebar">
+  <div class="mui--text-light mui--text-display1 mui--align-vertical">NEWS</div>
+  <% Category.all.each do |category| %>
```

Comment:
If we're iterating over the Categories already here to display them, why do we still manually list them below in the `h1` tags?

Reply:
thatäs a good poiny I will have a look at that